Catan_Game/Assets/Scripts/Game_Manager.cs

```diff
@@ -18,12 +18,16 @@ public class Game_Manager : MonoBehaviour
     Intersection[] intersections = Create_Intersections();
     int[,] tiles = Create_Tile_List();
 
-    // for future reference, order of the tokens
-    // 5 2 6 3 8 10 9 12 11 4 8 10 9 4 5 6 3 11
+    // dictionary that holds what intersection gets what yeild for a roll
+    // the key is the roll value while the arraylist will hold a touple that contains
+    // the intersection id and the yield
+    private Dictionary<int, ArrayList> roll_yeilds = new Dictionary<int, ArrayList>();
```

Comment:
yield*

Reply:
Whoops merged before doing this. Will fix in master